content/Mobile_Dev/WebAuthn/IOS/Register_Face_ID_and_Security_Keys.adoc

```diff
@@ -0,0 +1,119 @@
+= Register Face ID and Security Key on IOS Safari
+
+== I want to allow my users to register using both Face ID and a security key
+In this section we are going to walk through implementing an experience where a user is able to register a WebAuthn credential to their account using either Face ID or a security key. This will typically be the default experience in most applications as explicitly invoking Face ID or a security key requires the RP to explicitly note the option for creating the credential.
+
+If your user is using iOS 16, Face ID will create a copyable passkey. This credential will be synced across all of the users devices. While this has usability advantages for moving across devices, and is more secure than passwords, it’s not as secure as a device bound credential such as a security key.
+
+Security keys registered on iOS 16 will NOT create a credential that syncs across the users devices, and is bound to the hardware of the security key.
+
+=== Application demo
+Before we dive into the implementation guide, let’s first view an example of what we will be creating. The gif below demonstrates this scenario using our demo application.
+
+video::videos/reg_faceid_seckey_1.mp4[height=600]
+
+=== Invoking the WebAuthn registration ceremony
+To get started let's begin by refreshing ourselves on how to register a new credential with WebAuthn. Let's start with this basic account security screen. Here a user is given a list of their credentials, and a button that will begin the WebAuthn ceremony, allowing a user to add a new credential.
+
+Take a look at Figure 1 to see an example of an account security screen for adding a credential 
+
+image::images/reg_faceid_seckey_1.jpg[width=250]
+**Figure 1**
+
+Once the user is ready to register a new credential, the “Add a new WebAuthn credential” button is clicked.
+
+Figure 2 shows the modal that appears on the user's screen asking if they wish to use Face ID or their security key. 
+
+image::images/reg_faceid_seckey_2.jpg[width=500]
+**Figure 2**
+
+Figure 3 shows what happens when a user selects Face ID. A modal appears indicating that Face Id has been activated, and will attempt to scan your face. 
+
+image::images/reg_faceid_seckey_3.jpg[width=500]
+**Figure 3**
+
+Figure 4 shows what happens when a user selects Use Security Key. A modal appears asking the user to insert and activate their key. Subsequent prompts may include user verification, and a subsequent activation of the key to complete the registration process.
+
+image::images/reg_faceid_seckey_4.jpg[width=500]
+**Figure 4**
+
+Behind the scenes, the button is triggering a call to your relying party in order to retrieve the PublicKeyCreationOptions that are used to create the new credential.
```

Comment:
Consider mentioning why we get both FaceID and Security Key options here.  Because we didn't specify an authenticatorAttachment.

Reply:
Added in newest commit